### ISSUE TITLE
API documentation page fails to load when response representation has a "Definition" set

### ISSUE THREAD

**Issue by vincehorst** [Azure/api-management-developer-portal#425]
## Bug description

I've added a response representation sample for an API endpoint via the APIM in Azure Portal. If I elect to configure a Definition for the representation then the API endpoint's documentation page fails to load due to a JavaScript error.

* The failure occurs regardless whether I use an existing definition or create a new definition.
* The failure occurs regardless of the HTTP response code being configured (i.e. 200 vs 400, etc.)
* The only way to get the API endpoint documentation page to work again is to remove all sample responses; removing the Definition alone does not correct the issue.
* The failure occurs regardless of the Definition. For example, I tried creating a Definition with the content "{}" and it still causes a failure.

## Additional context
![image](https://user-images.githubusercontent.com/1152384/72573316-266c0380-387a-11ea-910d-4980a07d51bd.png)

(Updated screeshot ~10 minutes after opening the ticket - 2020-01-16 4:14 PM Pacific)
![image](https://user-images.githubusercontent.com/1152384/72573604-202a5700-387b-11ea-9884-ee29108183cf.png)

**Comment by ygrik** [comment]
this issue was fixed with commit #398

**Comment by vincehorst** [comment]
Hi @ygrik! I see that the commit #398 has been merged into the master branch on 1/8/2020. In practice, when could I expect this fix to arrive into my API Portal that is hosted by Microsoft in my Azure subscription? I am not able to find an online article describes what I can expect.

Also, once Microsoft updates their version, do I need to take any steps (e.g. to republish my portal)?

P.S. I republished my portal on 1/16/2020 and the issue was still present.

Thanks for your help!
